ops/model.py

```diff
@@ -1821,6 +1842,9 @@ def fetch(self, name: str) -> Path:
 
         If successfully fetched, this returns the path where the resource is stored
         on disk, otherwise it raises a :class:`NameError`.
+
+        Raises:
+            NameError: if the resource’s path cannot be fetched.
```

Comment:
It was intentional, mostly because I'd seen it used elsewhere and so assumed it was the style. VS Code highlights them:

![image](https://github.com/canonical/operator/assets/826522/ae4d0f8f-52e1-4a71-accc-a16cfa47a230)

so they are quite obvious (little yellow boxes scattered around).

I think the world is ready for non-ASCII docstrings for sure, and probably also comments. I'm less certain about non-ASCII names (except maybe if the code isn't English).

However, I wasn't aware that Sphinx automatically changes them. So with that in mind I'm happy to go with a plain `'`.